### ISSUE TITLE
Performance on SVHN dataset

### ISSUE THREAD

**Issue by jwyang** [soumith/dcgan.torch#25]
Hi Soumith, appreciate much for sharing the code.

Recently, I am trying to use your code to train dcgan on svhn. I tried many network architectures and hyperparameters, but failed to reproduce analogous performance as evaluated by the model shared by Alec Radford. What I can get is ~69% using 32x32 images and 1000 training labels. Unfortunately, I found Alec did not release the training code to get the model, so there is no reference to set up the network arch and determine the hyperparameters. So I would like to know whether you or any others have get a model that has similar performance to Alec's, say, 75+% accuracy on svhn based on 1000 labels.

thanks in advance!

**Comment by soumith** [comment]
@jwyang you could look at this repo which has very good svhn https://github.com/openai/improved-gan
